doc/en/user/source/extensions/wps-download/mapAnimationDownload.rst

```diff
@@ -215,7 +217,87 @@ The ``formattedTimestamper`` decoration ensures the frame time is included in th
       </decoration>
     </layout>
 
+Sample variable-based DownloadAnimation request
++++++++++++++++++++++++++++++++++++++++++++++++
 
+If you want to animate a map based on variables, proceed as follows:
+
+* specify the ``animateParam`` variable. You can use this to animate SLD variables, as described in ``Variable substitution in SLD``
```

Comment:
Dotted lists in English start with a capital letter and end with a full stop (unless the first word is a keyword that cannot be capitalized in any other way).